src/app/api/organizations/[organizationId]/route.ts

```diff
@@ -24,3 +27,43 @@ export async function DELETE(
   }
   return new NextResponse(undefined, { status: 204 });
 }
+
+// @route PUT /api/organizations/[organizationId] - updates an existing organization in the database
+export async function PUT(
+  _request: NextRequest,
```

Comment:
Missed this earlier, sorry. Pedantic, but remove the underscore and just call this parameter `request`. The `_` prefix is a convention that indicates that this is an unused param/variable